assets/src/application/ui/calendars/dateDisplay/CalendarDateSwitcher/index.tsx

```diff
@@ -12,8 +13,13 @@ const CalendarDateSwitcher: React.FC<CalendarDateSwitcherProps> = React.memo(
 	({ className, displayDate = DisplayStartOrEndDate.start, labels, ...props }) => {
 		const startDate = parseISO(props.startDate) || PLUS_ONE_MONTH;
 		const endDate = parseISO(props.endDate) || PLUS_TWO_MONTHS;
+
+		const footerPast: string | null = propOr<CalendarDateLabels>(null, 'footerPast')(labels);
+		const footerFuture: string | null = propOr<CalendarDateLabels>(null, 'footerFuture')(labels);
+		const footer: string = propOr<CalendarDateLabels>('', 'footer')(labels);
```

Comment:
@manzoorwanijk I have this weird TS error here, did you encounter such one?

![image](https://user-images.githubusercontent.com/1477453/78655845-34588000-78cf-11ea-8c00-23944d534601.png)

Reply:
If you want to go your way, this is what you can do:
```ts
const footerPast = propOr<string, CalendarDateLabels, string>(null, 'footerPast', labels);
const footerFuture = propOr<string, CalendarDateLabels, string>(null, 'footerFuture', labels);
const footer = propOr<string, CalendarDateLabels, string>('', 'footer', labels);
```

Reply:
thx, have gone with the other way